### PR TITLE
Model user

### PR DESCRIPTION
**Pull request by alisonreynoso**
## Summary
- Created the user model with its customer exception BookAlreadyOwned
## Trello Card
https://trello.com/c/2xwCDM1E/19-crear-modelo-de-user

### CODE REVIEW

File: src/main/java/wolox/training/models/User.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    private long id;
+    @Column(nullable = false)
+    private String username;
+    @Column(nullable = false)
+    private String name;
+    @Column(nullable = false)
+    private LocalDate birthdate;
+
+    @ManyToMany(cascade = CascadeType.ALL)
+    @JoinTable(name = "book_user",
```
Comment (julianvasquez-wolox) [inline src/main/java/wolox/training/models/User.java:33]: I don't know if this is really necessary but it seems good for me :D

**Review by julianvasquez-wolox — DISMISSED** [review]
Lets gooo

File: src/main/java/wolox/training/models/User.java (hunk 1 above)
Comment (Jimenarosello) [reply src/main/java/wolox/training/models/User.java:33]: I think it's not necessary 😄

File: src/main/java/wolox/training/models/User.java [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    public List<Book> getBooks() {
+        return (List<Book>) Collections.unmodifiableList(books);
+    }
+
+    public void setBooks(List<Book> books) {
+        this.books = books;
+    }
+
+    public void addBook(Book book) {
+        if (this.books.stream().anyMatch(book1 -> book1.getId() == book.getId())) {
```
Comment (Jimenarosello) [inline src/main/java/wolox/training/models/User.java:79]: Nice use of streams! 👏

**Review by Jimenarosello — DISMISSED** [review]
Great job! 😄

**Review by julianvasquez-wolox — APPROVED** [review]

**Review by Jimenarosello — APPROVED** [review]
🚀 😄

**Pull request merged**